site/pages/create-consequence-operator.tsx

```diff
@@ -0,0 +1,209 @@
+import Link from "next/link";
```

Comment:
This file will just need renaming to `create-consequence-operator.page.tsx`

Reply:
done